benchmarks/linear_algebra/blas/level1/asum/asum_generator.cpp

```diff
@@ -0,0 +1,81 @@
+
+#include <tiramisu/tiramisu.h>
+#include "benchmarks.h"
+#include "math.h"
+
+
+/*
+    ASUM:
+    ----
+    ASUM takes the sum of the absolute values.
+
+        SUM(ABS(X))
+
+    X is an ( 1 + ( N - 1 )*abs( incx ) ) vector    
+    incx storage spacing between elements of X
+        
+   
+    The C version of this function is as follow : 
+
+        result = 0;
+        for(int i=0; i < N :i++){
+	            result += x[i * incx]; 
+        }
+   
+*/
+
+
```

Comment:
Thanks! The code looks good to me, but I have one comment. The code does not respect the Tiramisu coding style. For example, the code should not have a new line when not necessary.

Reply:
Format the code structure of files:
syr_generator.cpp, syr_wrapper.cpp, asum_generator.cpp, asum_wrapper.cpp
to match the Tiramisu coding style.